misc/userscripts/qutebrowser_viewsource

```diff
@@ -24,9 +24,15 @@
 # Caveat: Does not use authentication of any kind. Add it in if you want it to.
 #
 
+TERMINAL=${TERMINAL:-urxvt}
+EDITOR=${EDITOR:-vim}
+
 path=/tmp/qutebrowser_$(mktemp XXXXXXXX).html
+curl "$QUTE_URL" > "$path"
 
-curl "$QUTE_URL" > $path
-urxvt -e vim "$path"
+unset _command
+[[ $EDITOR =~ emacs|gedit|leafpad|atom ]] && _command="$EDITOR $path"
+[ -z "$_command" ] && _command="$TERMINAL -e \"vim $path\""
 
+eval "$_command"
```

Comment:
Thinking about it again, maybe we should just keep it hardcoded to urxvt, and make the user copy/adjust the userscript if they want to use something else.